shared/ui/inputValidators/validateNumericInput.ts

```diff
@@ -0,0 +1,46 @@
+import { Zero } from '@ethersproject/constants';
+import { BigNumber } from 'ethers';
+import { formatEther, parseEther } from '@ethersproject/units';
+
+export const validateNumericInput = (
+  value: string,
+  inputName: string,
+  {
+    maximum,
+    minimum,
+  }: {
+    maximum?: BigNumber;
+    minimum?: BigNumber;
+  } = {},
+): string | true => {
+  const amount = Number(value);
```

Comment:
parseInt/parseFloat is more explicit

Reply:
parseInt/parseFlot work differently, feels like it's better to get NaN here than try to actually parse number

<img width="305" alt="image" src="https://github.com/lidofinance/trp-ui/assets/103929444/c7497c6a-406a-46d9-b553-35c9541be5d7">